### PR TITLE
More details on log4j 1.2

### PR DESCRIPTION
**Pull request by madrob**

### CODE REVIEW

File: content/solr/security/2021-12-10-cve-2021-44228.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Critical
 **Description:**
 Apache Solr releases prior to 8.11.1 were using a bundled version of the Apache Log4J library vulnerable to RCE. For full impact and additional detail consult the Log4J security page.
 
-Apache Solr releases prior to 7.0 (i.e. all Solr 5 and Solr 6 releases) use log4j 1.2.17 which may be vulnerable for installations using non-default logging configurations. To determine you if you are vulnerable please consult the Log4J security page.
+Apache Solr releases prior to 7.4 (i.e. Solr 5, Solr 6, and Solr 7 through 7.3) use log4j 1.2.17 which may be vulnerable for installations using non-default logging configurations that include the JMS Appender, see https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126 for discussion.
```
Comment (uschindler) [inline content/solr/security/2021-12-10-cve-2021-44228.md:14]: Please put links inside `<http...>` to make them clickable.

**Review by uschindler — COMMENTED** [review]
Link is not clickable:

File: content/solr/security/2021-12-10-cve-2021-44228.md (hunk 1 above)
Comment (uschindler) [reply content/solr/security/2021-12-10-cve-2021-44228.md:14]: Check the other links I already fixed earlier.

Comment (thelabdude) [reply content/solr/security/2021-12-10-cve-2021-44228.md:14]: done: https://github.com/apache/solr-site/commit/7eba60ca07f0215b2f616513cbf0756ee715dd18

**Comment by uschindler** [comment]
Hi,
According to newest information in the linked issue, log4j v1 ist NOT vulnerable. It can execute jndi links, but not execute Code as no java serialization is possible.

**Comment by GameScripting** [comment]
@uschindler does this mean, that it suffers from the [data-exfiltration issue](https://github.com/apache/logging-log4j2/pull/608#issuecomment-991354707)?

**Pull request merged**